AnalyticalModels/data/README.md

```diff
@@ -0,0 +1,28 @@
+The coefficients calculated bin by bin are in "coefficientsByBin_klkt.txt"
+
+The bins are defined in the gen-level variables as:
+
+binGenMHH = [250.,270.,300.,330.,360.,390., 420.,450.,500.,550.,600.,700.,800.,1000.] 
+binGenCostS  = [ -1., -0.55,0.55,1.  ] 
+
```

Comment:
could you put these two lines in a code environment so that it gets formatted properly ?